guide/14-deep-learning/doccano setup guide.ipynb

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
<p>General Suggestions: </p><p> </p><p>a) Images can be placed above text in each of the steps.</p><p>b) Full stop after every step. For instance, steps 3, 8, 9, 10 and 11.</p><p> </p><blockquote>Labelling data for training with Doccano.</blockquote><p> </p><p>Suggestion: No full stop required. Title can be reframed. Can be changed to "<strong>Labeling training data using Doccano</strong>"</p><p> </p><blockquote>Create new project with project type 'Sequence labelling'</blockquote><p>Suggestion: Spelling of labelling is different in text and image. Can be changed as per image to ''Sequence labelling".</p><p> </p><blockquote>Click on import data</blockquote><p>Suggestion: Can be changed to: To import data for annotation, click on '<strong>Import Data</strong>'.</p><p> </p><blockquote>Click on 'Annotate Data' from the top menu bar.</blockquote><p>Suggestion: Can be changed to "Click on 'Annotate Data' from the top menu bar to annotate the data."</p><p> </p><blockquote>Select 'JSONL' and then click on 'Select a file' and point it to the reports file (docanno_deployment_scripts\reports_labels.json). Alternatively, text documents can also be uploaded using the ‘plain’ option. You can either upload one text document at a time or upload a single text file with one document per line.</blockquote><p> </p><p>Suggestion: Please check the name of the json file. Is it reports_labels.json or reports_label.json.</p><p> </p><blockquote>All the documents are pre-labelled, just 3 (document number 2,3 &amp; 4) are intentionally left unlabeled for you to try labelling. Analyze the first labelled document and then move on to second document (use the bottom navigation bar for sifting through the docs). Mark sequences with your mouse and select the relevant title.</blockquote><p> </p><p>Suggestions:</p><p> </p><p>a) Please make spelling of labelled consistent throughout.</p><p>b) Kindly check whether the documents 2, 3 and 4 in the specified file are unlabeled.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/644/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>